src/managers/WebhookManager.js

```diff
@@ -37,23 +38,111 @@ class WebhookManager {
   _onNewWebhookEvent = (webhook: Webhook): (event: Event) => void =>
     (event: Event) => {
       try {
-        const responseHandler = (error, response) => {
+        const defaultWebhookVariables = {
+          // todo add old defaults for compatibility
+          PARTICLE_DEVICE_ID: event.deviceID,
+          PARTICLE_EVENT_NAME: event.name,
+          PARTICLE_EVENT_VALUE: event.data,
+          PARTICLE_PUBLISHED_AT: event.publishedAt,
+        };
+
+        let eventDataVariables = {};
+        if (typeof event.data === 'string') {
+          try {
+            eventDataVariables = JSON.parse(event.data);
+          } catch (error) {
+            eventDataVariables = {};
```

Comment:
So does everything work if `event.data` is equal to `12345`?  Because you can't guarantee that the user will always pass JSON from the device.  In that case, the data should just be forwarded directly to the webhook server request.